### PR TITLE
QuakeML web services.

### PR DESCRIPTION
**Pull request by gclitheroe**
sc3ml-to-quakeml provides web services for QuakeML and CSV for events.

* It replaces https://github.com/GeoNet/seiscompml-to-quakeml
* Particularly the routes at https://github.com/GeoNet/seiscompml-to-quakeml/blob/master/.ebextensions/httpd.conf#L389

All of the returned content is created using XSLT.  The hard work happens in `func xslt(...)`, everything else is just request wiring.

Uses Alpine for Docker so that it can install xsltproc (from libxslt).  @quiffman - would be appreciated if you could have a look at the Docker file as well (`sc3ml-to-quakeml`). 

All the failed builds are me force pusing trying to get Travis to install xsltproc :-(

### CODE REVIEW

File: sc3ml-to-quakeml/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM alpine:latest
```
Comment (quiffman) [inline sc3ml-to-quakeml/Dockerfile]: I've been leaning back towards using specific versions in the FROM statement.
It seems more deterministic, particularly from a CI/CD build-system perspective.

File: sc3ml-to-quakeml/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+RUN apk update \
```
Comment (quiffman) [inline sc3ml-to-quakeml/Dockerfile]: They recommend using, `RUN apk add --no-cache libxslt` now.

File: sc3ml-to-quakeml/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    && apk add libxslt \
+    && apk upgrade
```
Comment (quiffman) [inline sc3ml-to-quakeml/Dockerfile]: I don't _think_ `apk upgrade` is strictly necessary.
But feel free to keep it if you want.

**Comment by crleblanc** [comment]
Looks good, nice to see Go has good support for running command line programs (not surprising).  All the right stdout/stderr/exit status stuff, surprisingly difficult to do properly in Python!

**Pull request merged**